exe_14.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+
+def combine_arrays(vector:int, array:np):

```

Comment:
Use more specific types for type hinting.
https://numpy.org/devdocs/reference/typing.html

I dont' think vector is of type int.